api/jobs_test.go

```diff
@@ -260,6 +260,291 @@ var _ = Describe("Jobs API", func() {
 		})
 	})
 
+	Describe("GET /api/v1/pipelines/:pipeline_name/jobs/:job_name/badge", func() {
+		var response *http.Response
+
+		JustBeforeEach(func() {
+			var err error
+
+			response, err = client.Get(server.URL + "/api/v1/pipelines/some-pipeline/jobs/some-job/badge")
+			Expect(err).NotTo(HaveOccurred())
+
+			Expect(pipelineDBFactory.BuildWithTeamNameAndNameCallCount()).To(Equal(1))
```

Comment:
Could these expectations be moved out of the JustBeforeEach? It seems like they're actual test expectations rather than setup.